### PR TITLE
Add CoC configuration

### PR DESCRIPTION
**Pull request by metal3d**
The CoC configration is set in coc-settings.json. The added section explains how to configure helm_ls with CoC plugin.

### CODE REVIEW

**Comment by metal3d** [comment]
Related to #76

File: README.md [hunk 1]
```diff
@@ -178,6 +178,33 @@ settings = {
...
+
+Open Neovim and type the command `:CocConfig` to access the configuration file. Find the `langageserver` section, or create one if necessary, and add this configuration:
+
+
+```json
+"languageserver": {
+  "helm": {
+    "command": "helm_ls",
+    "args": ["serve"],
+    "filetypes": ["yaml", "helm", "helmfile"],
```
Comment (qvalentin) [inline README.md:195]: Im not sure about having yaml here.

How did you setup the filetypes helm and helmfile in your config? With towolf/vim-helm?

File: README.md [hunk 2]
```diff
@@ -178,6 +178,33 @@ settings = {
 See [examples/nvim/init.lua](https://github.com/mrjosh/helm-ls/blob/master/examples/nvim/init.lua) for an
 complete example, which also includes yaml-language-server.
 
+#### Neovim + CoC plugin
+
+Using the [coc.nvim](https://github.com/neoclide/coc.nvim) plugin, you don't need the previous configuration in lua. Coc will handle the configuration instead.
```
Comment (qvalentin) [inline README.md:183]: > you don't need the previous configuration in lua

Maybe we should restructure the readme to something like:

### Neovim config
explain filetypes etc (anything that must be done for both nvim-lspconfig and coc)

#### nvim-lspconfig
explain setup with nvim-lspconfig
#### coc
explain setup with coc

**Review by qvalentin — COMMENTED** [review]
Thanks for the contribution.
I just have some small questions.

File: README.md (hunk 1 above)
Comment (metal3d) [reply README.md:195]: Yes right.

Maybe I can remove yaml from the list. I will do it today.

File: README.md (hunk 2 above)
Comment (mrjosh) [reply README.md:183]: any update on this?

**Pull request closed without merging**